setup.py

```diff
@@ -11,7 +11,10 @@ class zipcodetw_install(install):
     def run(self):
         print('Building ZIP code index ... ')
         sys.stdout.flush()
-        zipcodetw.builder.build()
+        try:
+            zipcodetw.builder.build()
+        except:
```

Comment:
This should catch only sqlite's exception .

A better approach is to fix the code inside .builder